### PR TITLE
Fix-notifications-endpoint

### PR DESCRIPTION
**Pull request by Stezido**
1. The **schema** of the notification.list endpoint has to be updated so the response contains all fields again.
2. Live notifications on the notification page fetch wrong count of new notifications. The notification list is not instantly updated when fetching live updates so the **live update can be disabled** on the notification page

### CODE REVIEW

**Comment by Stezido** [comment]
Closes #182 
Closes #183

File: frontend/src/pages/Notifications/NotificationPageContainer.js [hunk 1]
```diff
@@ -35,7 +43,9 @@ const mapDispatchToProps = (dispatch, props) => {
     markMultipleNotificationsAsRead: (notificationIds, offset, limit) =>
       dispatch(markMultipleNotificationsAsRead(notificationIds, offset, limit)),
     setNotifcationsPerPage: limit => dispatch(setNotifcationsPerPage(limit)),
-    setNotificationOffset: offset => dispatch(setNotificationOffset(offset))
+    setNotificationOffset: offset => dispatch(setNotificationOffset(offset)),
+    enableLiveUpdates: isEnabled => dispatch(enableLiveUpdates(isEnabled)),
```
Comment (mathiashoeld) [inline frontend/src/pages/Notifications/NotificationPageContainer.js]: `enableLiveUpdates` --> `setIsLiveUpdatesEnabled`?

File: frontend/src/pages/Notifications/actions.js [hunk 2]
```diff
@@ -124,3 +126,10 @@ export function setNotificationOffset(offset) {
     offset
   };
 }
+
+export function enableLiveUpdates(isEnabled) {
```
Comment (mathiashoeld) [inline frontend/src/pages/Notifications/actions.js]: change here too

File: frontend/src/pages/Notifications/reducer.js [hunk 3]
```diff
@@ -37,6 +39,10 @@ export default function navbarReducer(state = defaultState, action) {
         notificationCount: action.notifications.length
       });
 
+    case SET_LIVE_UPDATES: {
```
Comment (mathiashoeld) [inline frontend/src/pages/Notifications/reducer.js]: rename?

File: frontend/src/pages/Notifications/NotificationPageContainer.js (hunk 1 above)
Comment (Stezido) [reply frontend/src/pages/Notifications/NotificationPageContainer.js]: I split the action into enable/disable live Updates.

**Pull request merged**